lib/java/com/google/android/material/internal/CollapsingTextHelper.java

```diff
@@ -699,30 +690,27 @@ private boolean shouldDrawMultiline() {
   }
 
   private void drawMultinlineTransition(
-      @NonNull Canvas canvas, float currentExpandedX, float x, float y, float ascent) {
+      @NonNull Canvas canvas, float currentExpandedX, float y) {
     int originalAlpha = textPaint.getAlpha();
     // positon expanded text appropriately
     canvas.translate(currentExpandedX, y);
     // Expanded text
     textPaint.setAlpha((int) (expandedTextBlend * originalAlpha));
     textLayout.draw(canvas);
 
-    // position the overlays
-    canvas.translate(x - currentExpandedX, 0);
```

Comment:
Here I admit, that I am not sure what was this line supposed to do, so this might not be the smartest thing to do. 

However, we want to draw the single line in the same place as the multiline above via `textLayout.draw(canvas)`, so moving away from that point does not seem right.